pytorch_lightning/core/lightning.py

```diff
@@ -312,18 +312,7 @@ def log(
 
             self.log('train_loss', loss)
 
-        The default behavior per hook is as follows:
-
-        .. csv-table:: ``*`` also applies to the test loop
-           :header: "LightningModule Hook", "on_step", "on_epoch", "prog_bar", "logger"
-           :widths: 20, 10, 10, 10, 10
-
-           "training_step", "T", "F", "F", "T"
-           "training_step_end", "T", "F", "F", "T"
```

Comment:
I believe this one was quite useful. Helped me to understand the default quite fast.

Reply:
This is added to the docs now. Because we have more hooks that are not mentioned here, so I moved them inside docs and linked them to the API.

Reply:
here: https://140072-178626720-gh.circle-artifacts.com/0/html/extensions/logging.html#id4